make/test.mk

```
@@ -7,8 +7,12 @@
 QUAY_NAMESPACE ?= codeready-toolchain
 DATE_SUFFIX := $(shell date +'%d%H%M%S')
 RESOURCES_SUFFIX := ${DATE_SUFFIX}
-MEMBER_NS := ${QUAY_NAMESPACE}-member-${DATE_SUFFIX}
+ifeq ($(HOST_NS),) # unless it's already defined as an env var
 HOST_NS := ${QUAY_NAMESPACE}-host-${DATE_SUFFIX}
+endif
+ifeq ($(MEMBER_NS),) # unless it's already defined as an env var
+MEMBER_NS := ${QUAY_NAMESPACE}-member-${DATE_SUFFIX}
+endif
```

I think that instead of this complex if statements you can use the "lazy if absent assignment"
```suggestion
HOST_NS ?= ${QUAY_NAMESPACE}-host-${DATE_SUFFIX}
MEMBER_NS ?= ${QUAY_NAMESPACE}-member-${DATE_SUFFIX}
```

---

ah thanks! I wasn't sure about the best way to implement this, and I forgot about the `?=` operator to lazily initialize the variables 👍